.circleci/config.yml

```diff
@@ -310,14 +310,9 @@ defaults:
         CXX: clang++
         MAKEFLAGS: -j 3
 
-  - base_ubuntu2004_clang_small: &base_ubuntu2004_clang_small
+  - base_ubuntu2004_clang_medium: &base_ubuntu2004_clang_medium
       <<: *base_ubuntu2004_clang
-      resource_class: small
-      environment:
-        TERM: xterm
-        CC: clang
-        CXX: clang++
```

Comment:
It's no longer really `base_ubuntu2004_`**clang**`_medium` if you remove the clang part :P